.github/workflows/ci.yml

```diff
@@ -0,0 +1,107 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🛑 Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: ⬇️ Checkout
+        uses: actions/checkout@v2
+
+      - name: ⎔ Setup node for problem matcher
+        if: github.event.pull_request
+        uses: actions/setup-node@v2
+
+      - name: 📥 Download deps
+        run: npm install
+
+      - name: 🧪 Run lint
+        run: npm run lint
+
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    needs: lint
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - 12
```

Comment:
```suggestion
          - 12.22.0
```

Reply:
You can see if these version bumps are really necessary to pass the tests, but IIRC some downstream tools at one point at least were going to need this version, or possibly some of the linting or testing wouldn't work without the changes.